### PR TITLE
Couple of changes to GUIs

### PR DESCRIPTION
**Pull request by Moonlight63**
I want to have all the GUIs working and ready to go before I re-add localization. I have a couple more changes in mind though.

### CODE REVIEW

**Comment by jmc2obj** [comment]
I'm still having some problems adjusting to Git. I'm not sure how to add changes to your pull request (if that is even possible), so I made a new branch out of it (tmp91) and added some small changes there. Do you have any advice wrt that?

**Comment by Moonlight63** [comment]
I don't think you can make changes to an incoming pull request. the only thing you could do is make a pull request on my fork. BTW, the only reason I made a fork is because I can't commit directly to this repo. I think its a permissions thing. Otherwise I would just add to this repo without the need of a pull request, and then any changes anyone else makes would be automatically updated in my eclipse workspace, which is really how it's "supposed" to work.

in any case, I am making a new pull right now with some more changes.

**Comment by danijel3** [comment]
Okay. I will keep merging your requests then, as that will make things simpler.

However, I think you should be able to push directly to the repo. You are added as a collaborator and it says you have push permissions on. Is there anything I should change in the repo settings?

**Comment by Moonlight63** [comment]
hmm, let me try again real quick

**Comment by Moonlight63** [comment]
yea, it seems I don't have write permissions. Am I set to have read-write access?

**Comment by danijel3** [comment]
Yep, you're right!

I added you now. Got you completely confused with someone else! Man I feel stupid now.

You should be able to do everything normally now :)

**Comment by Moonlight63** [comment]
Yep, we're good now, lol
